src/pages/workspaces/List.js

```diff
@@ -356,8 +364,13 @@ export const WorkspaceList = () => {
           href: 'https://support.terra.bio/hc/en-us/articles/360024743371-Working-with-workspaces'
         }, ['Learn more about workspaces.'])
       ]),
-      !_.isEmpty(workspaces) && !_.isEmpty(recentlyViewed) && div([
-        p({ style: { textTransform: 'uppercase' } }, 'Recently viewed'),
+      !_.isEmpty(workspaces) && !_.isEmpty(recentlyViewed) && h(Collapse, {
+        title: 'Recently Viewed',
+        initialOpenState: recentlyViewedOpen !== undefined ? recentlyViewedOpen : true,
```

Comment:
If an undefined local preference is handled in the state initialization, then this becomes
```suggestion
        initialOpenState: recentlyViewedOpen,
```